### PR TITLE
Add metadataUrls to CatalogMember

### PR DESCRIPTION
**Pull request by nf-s**
### Add metadataUrls to CatalogMemberTraits

Fixes https://github.com/TerriaJS/oceanexplorer/issues/10
Add `metadataUrls` trait to `CatalogMemberTraits` of form:

```json
metadataUrls": [
  {
    "title": "MARLIN Metadata",
    "url": "http://www.somemetadata.com"
  }
]
```

http://ci.terria.io/metadataurls/#clean&https://gist.githubusercontent.com/nf-s/d10b07b07cba606a2a60a9631517d0ca/raw/cdd64a965fe2b9f64bbdfa7face9344122ae416a/marlin.json

Looks like:

image

Or, if `title` is undefined, it looks like:

image

### Checklist

-   [x] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
-   [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

**Comment by nf-s** [comment]
Test catalog https://gist.githubusercontent.com/nf-s/d10b07b07cba606a2a60a9631517d0ca/raw/2444ce25577090d6d04333347b0033178af21b99/marlin.json

File: lib/ReactViews/Preview/Description.jsx [hunk 1]
```diff
@@ -180,23 +183,31 @@ const Description = observer(
...
-                </a>
-              </p>
+            <If
+              condition={
+                catalogItem.metadataUrls && catalogItem.metadataUrls.length > 0
+              }
+            >
+              <h4 className={Styles.h4}>{t("description.metadataUrls")}</h4>
+              <For each="metadataUrl" index="i" of={catalogItem.metadataUrls}>
+                <Box paddedVertically>
```
Comment (na9da) [inline lib/ReactViews/Preview/Description.jsx]: There is a console warning for not passing a key for the list child, add a `key={metadataUrl.url}` for the `Box` here?

Comment (nf-s) [reply lib/ReactViews/Preview/Description.jsx]: Done!

**Review by na9da — APPROVED** [review]
Looks great @nf-s .

The only other issue I could think of is if the catalog definitions contains extra title definitions, then we will be showing buttons with empty links. So maybe we could filter out buttons with empty links by wrapping this in an 'IF`.

https://github.com/TerriaJS/terriajs/blob/f5f55af1c2563e427a6192302f873487e2eef9f0/lib/ReactViews/Preview/Description.jsx#L193-L208

**Pull request merged**